pypingdom/api.py

```diff
@@ -9,7 +9,7 @@
 class ApiError(Exception):
 
     def __init__(self, http_response):
-        content = json.loads(http_response.content)
+        content = json.loads(http_response.content.decode('utf-8'))
```

Comment:
Maybe it would be better to use a requests feature as documented in http://www.python-requests.org/en/latest/user/quickstart/#json-response-content:

`content = http_response.json()`